dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -421,22 +421,22 @@ ANNO_TARGETS_CORRUPT_CLASS.useraction=The class can be replaced, removed or this
 # # {2} The error message from the exception
 ANNO_CLASSSOURCE_CHILD_OPEN_EXCEPTION=CWWKC0065W: [ {0} ]: The open of child class source [{1}] caused an exception. Error message: {2}
 ANNO_CLASSSOURCE_CHILD_OPEN_EXCEPTION.explanation=The open of the child class source caused an exception.
-ANNO_CLASSSOURCE_CHILD_OPEN_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_CLASSSOURCE_CHILD_OPEN_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
```

Comment:
Change
'For more information on the topic, see the exception and the associated FFDC message.'
to
'For more information, see the exception and the associated FFDC message.'

Reply:
Updated.